doc/zypper.8.txt

```diff
@@ -455,6 +455,19 @@ include::{incdir}/option_Solver_Flags_Common.txt[]
 		No automatic removal of unneeded dependencies.
 --
 
+*purge-kernels* ['options']::
+	Autoremoves installed kernels.
+	+
+	Automatically cleans up installed kernels according to the rules defined in /etc/zypp/zypp.conf:multiversion.kernels which can be given as <version>, latest(-N), running, oldest(+N).
```

Comment:
The AFAICS most common way of referring to a zypp.conf entry in the man page is `[zypp.conf:multiversion.kernels]`.

Reply:
Done